apps/web/pages/[user].tsx

```diff
@@ -204,14 +188,14 @@ export default function User(props: inferSSRProps<typeof getServerSideProps> & E
               ))
             )}
           </div>
+
           {isEventListEmpty && <EmptyPage name={user.name ?? "User"} />}
         </main>
         <Toaster position="bottom-right" />
       </div>
     </>
   );
 }
-User.isThemeSupported = true;
```

Comment:
Is this unsupported?

Reply:
User.isThemeSupported = true 
and 
undefined 
means we enable dark/light mode

Component.isThemeSupported = false -> we force light mode